src/mod_manager.cpp

```diff
@@ -149,6 +150,13 @@ bool mod_manager::has_mod(const std::string &ident) const
     return mod_map.count(ident) > 0;
 }
 
+void mod_manager::load_user_mods_from(std::string path)
+{
+  for( auto &mod_file : get_files_from_path(MOD_SEARCH_FILE, path, true) ) {
+      load_mod_info( mod_file );
+  }
+}
+
 void mod_manager::load_mods_from(std::string path)
 {
     for( auto &mod_file : get_files_from_path(MOD_SEARCH_FILE, path, true) ) {
```

Comment:
Considering that the following two calls to `load_mod_info` do *not* depend on the function parameter `path`, they could be moved directly after where `load_mods_from` is calls (only one place, in `refresh_mods_list`). Than you could call this function directly instead of having a new one.

Reply:
I will fix this when i get off work, thanks for the suggestion.